### DISCUSSION TITLE
Did K3S just used my port 80 and 443?

### DISCUSSION THREAD

**Discussion by stevefan1999-personal** [k3s-io/k3s#5261] [Q&A]
I have a question why these two ports are being used by k3s:
```
root@serv:~# netstat -lnp | grep :80
tcp        0      0 192.168.2.10:80         0.0.0.0:*               LISTEN      1020795/k3s server

root@serv:~# netstat -lnp | grep :443
tcp        0      0 192.168.2.10:443        0.0.0.0:*               LISTEN      1020795/k3s server
```
This seemingly caused my Calico to not function correctly...All I got is just connection refused despite it is listening. I checked that this process is also sleeping, hence, so my theory of why it is rejecting is that it never responded to the SYN

**Comment by brandond** [comment]
https://rancher.com/docs/k3s/latest/en/networking/#traefik-ingress-controller

> The Traefik ingress controller will use ports 80 and 443 on the host (i.e. these will not be usable for HostPort or NodePort).

**Reply by stevefan1999-personal** [reply]
I have disabled it prior, I use my own nginx ingress

**Reply by brandond** [reply]
Ah, that would be good to mention in describing your issue. K3s does not use those ports itself, so if you've disabled traefik, it would be related to something you've deployed.
